src/main/kotlin/no/nav/syfo/identhendelse/kafka/IdenthendelseConsumerService.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.syfo.identhendelse.kafka
+
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import no.nav.syfo.application.ApplicationState
+import no.nav.syfo.identhendelse.IdenthendelseService
+import no.nav.syfo.kafka.KafkaConsumerService
+import no.nav.syfo.metric.COUNT_KAFKA_CONSUMER_PDL_AKTOR_TOMBSTONE
+import org.apache.avro.generic.GenericData
+import org.apache.avro.generic.GenericRecord
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+class IdenthendelseConsumerService(
+    private val applicationState: ApplicationState,
+    private val identhendelseService: IdenthendelseService,
+) : KafkaConsumerService<GenericRecord> {
+    override val pollDurationInMillis: Long = 1000
+
+    override fun pollAndProcessRecords(kafkaConsumer: KafkaConsumer<String, GenericRecord>) {
+        runBlocking {
```

Comment:
Her hadde vi en `coroutinescope` i `isdialogmote`, men ettersom det var et annet oppsett her med `KafkaConsumerService`-interfacet og override av `pollAndProcessRecords`, så prøvde jeg meg på en `runBlocking` i stedet for å gjøre alle `pollAndProcessRecords`-funksjonene til `suspend`-funksjoner. 

Noen tanker?

Reply:
Tror det bør gå bra siden denne consumeren kjører i sin egen tråd.

Reply:
Det var lurt, Geir! 🥇

Reply:
🧠 📈

Reply:
Eller vi har jo denne `delay()` funksjonen som kjører retries hvert 60. sekund, så den må være inne i en sånn coroutinescope.